docs/options.md

```diff
@@ -0,0 +1,30 @@
+# Changing PostgreSQL Options
+
+You may require a configuration change for your application. PostgreSQL
+allows the option to configure the database with a configuration file, as many other database
+management systems do. You can pass options to PostgreSQL instances using the
+specific Custom Resource option via the `deploy/cr.yaml` configuration file.
+
+Often there's no need to add custom options, as the Operator takes care of
+providing PostgreSQL with reasonable defaults.
```

Comment:
Do we have any auto tuning in place that we say this? If so - is it described anywhere?
If not - please remove this.